featuretools/primitives/standard/transform_primitive.py

```diff
@@ -653,8 +653,8 @@ class Lag(TransformPrimitive):
         You can specify the number of periods to shift the values
 
         >>> lag_periods = Lag(periods=3)
-        >>> lag_periods(["hello", "world", "test", "foo", "bar"], pd.Series(pd.date_range(start="2020-01-01", periods=5, freq='D'))).tolist()
-        [nan, nan, nan, 'hello', 'world']
+        >>> lag_periods([True, False, False, True, True], pd.Series(pd.date_range(start="2020-01-01", periods=5, freq='D'))).tolist()
+        [nan, nan, nan, True, False]
     """
```

Comment:
was something wrong with the strings?

Reply:
Not wrong exactly, but complicated - using strings this way with the new version of pandas introduced `None` instead of `nan`, which would mean this docstring test would pass with old versions of pandas but not version `1.5.0`.

Also based on the specified input types we wouldn't really be calling this function with a `object` type input like we were doing with the string input. So, in order to allow this test to pass on old and new versions of pandas, I switched to an input that is in our supported input types and gives consistent output between pandas versions.